### ISSUE TITLE
Favicon

### ISSUE THREAD

**Issue by wintermeyer** [wort-schule/wort.schule#258]
Die Ecken des Favicons sind in Safari nicht korrekt. Nicht gravierend. Es fehlt eine Kleinigkeit.

<img width="87" alt="Bildschirm­foto 2023-03-07 um 09 36 54" src="https://user-images.githubusercontent.com/69770/223368240-07c46b5a-46f5-458e-87cf-d4a74420f30c.png">

@cabotine Ruhig das Issue schliessen, wenn Du es als unwichtig betrachtest.

**Comment by cabotine** [comment]
Nee. Es nervt mich auch schon die ganze Zeit bei Firefox im Dunkelmodus... ich versuche es mal zu fixen...
